src/components/Episode/styles.scss

```diff
@@ -1,7 +1,31 @@
 .wrapper {
-	font-size: 1em; //placeholder styling
-
+	margin-top: 20vh;
```

Comment:
Is this a margin on the top of this component, or a padding on the top of its parent? Haven't checked, I'm just asking in-case you haven't considered it yourself yet!

Reply:
Yeah that does make more sense for it to be padding-top for parent 👍